apm-agent-plugins/apm-es-restclient-plugin/apm-es-restclient-plugin-7_1/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>apm-es-restclient-plugin</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.6.2-SNAPSHOT</version>
```

Comment:
```suggestion
        <version>1.7.1-SNAPSHOT</version>
```

Reply:
@kananindzya please just update the version so we can commit.